README.md

```diff
@@ -1,2 +1,8 @@
-# Cytotoxic-Nuisance-Metadata-Analysis
-Metadata search within the Cytotoxic and Nuisance Metadata
+# Metadata Search
+
```

Comment:
Consider adding a goal here. What is the purpose of the metadata and what will you do with it?